sdcm/nemesis.py

```diff
@@ -1211,6 +1211,9 @@ def disrupt_nodetool_decommission(self, add_node=True, disruption_name=None):
                 self.unset_current_running_nemesis(new_node)
         return new_node
 
+    def disrupt_nodetool_decommission(self, add_node=True, disruption_name=None):
+        return self._nodetool_decommission(add_node=add_node, disruption_name=disruption_name)
```

Comment:
`disruption_name` was needed only cause `disrupt_nodetool_seed_decommission` was calling this nemesis function

```suggestion
    def disrupt_nodetool_decommission(self, add_node=True):
        return self._nodetool_decommission(add_node=add_node)
```

in general out disrupt function shouldn't have parameters, and shouldn't be calling each other directly

Reply:
fixed